### ISSUE TITLE
Compilation Message Update

### ISSUE THREAD

**Issue by Xecute0101** [qwertycoin-org/qwertycoin#108]
"/home/user/.hunter/_Base/bc7da32/7f32107/75298a5/Install/include/boost/pending/integer_log2.hpp:7:89: note: #pragma message: This header is deprecated. Use <boost/integer/integer_log2.hpp> instead."

Above msg keeps popping on Linux 16/18. Can hunter be updated to use the file in recommendation instead?

**Comment by exploshot** [comment]
Fixed by 469d63ceff1b1fb417826c2a7e1f4cd3e55cd984 and 6b13eda36a9c11c48ab2f55294597dc9a77b643d

I've Updated boost to 1.70.

**Comment by Xecute0101** [comment]
[ 59%] Building CXX object lib/CMakeFiles/QwertycoinFramework_Wallet.dir/WalletLegacy/WalletTransactionSender.cpp.o

In function ‘T CryptoNote::{anonymous}::popRandomValue(URNG&, std::vector<T>&) [with URNG = std::linear_congruential_engine<long unsigned int, 16807, 0, 2147483647>; T = long unsigned int]’:

cc1plus: warning: ‘void* __builtin_memset(void*, int, long unsigned int)’: specified size 18446744073709551608 exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]

**Comment by Xecute0101** [comment]
config.status: src/config.h is unchanged
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_SparseHashMap<int, int, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_SparseHashMap<int, int, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_SparseHashSet<int, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_SparseHashSet<int, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_SparseHashtable<int, int, {anonymous}::Hasher, {anonymous}::Negation<int>, {anonymous}::SetKey<int, {anonymous}::Negation<int> >, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_SparseHashtable<int, int, {anonymous}::Hasher, {anonymous}::Negation<int>, {anonymous}::SetKey<int, {anonymous}::Negation<int> >, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_DenseHashMap<int, int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_DenseHashMap<int, int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_DenseHashSet<int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_DenseHashSet<int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~
src/hashtable_test.cc: In instantiation of ‘void {anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::Run() [with TypeParam = google::HashtableInterface_DenseHashtable<int, int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Negation<int>, {anonymous}::SetKey<int, {anonymous}::Negation<int> >, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’:
src/hashtable_test.cc:571:1:   required from ‘{anonymous}::TEST_onetype_HashtableIntTest_Typedefs<TypeParam>::TEST_onetype_HashtableIntTest_Typedefs() [with TypeParam = google::HashtableInterface_DenseHashtable<int, int, kEmptyInt, {anonymous}::Hasher, {anonymous}::Negation<int>, {anonymous}::SetKey<int, {anonymous}::Negation<int> >, {anonymous}::Hasher, {anonymous}::Alloc<int> >]’
src/hashtable_test.cc:571:1:   required from here
src/hashtable_test.cc:577:32: warning: variable ‘kt’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::key_type kt;
                                ^~
src/hashtable_test.cc:583:33: warning: variable ‘st’ set but not used [-Wunused-but-set-variable]
   typename TypeParam::size_type st;
                                 ^~

* In short, variable kt and st are specified but never used.
